queries/highlights.scm

```diff
@@ -134,6 +135,13 @@
   "clear"
 ] @keyword)
 
+;; Command command
+
+(command_attribute name: _ @property)
+(command_attribute
+  value: (behavior
+    (identifier)? @function) @constant)
```

Comment:
It is not a always `function` here, right ? Could be an integer or something else.

Reply:
The only time there is a node inside a behavior is either for `custom` or `customList` which only accepts function name after the comma